src/pages/QuizDetails.vue

```diff
@@ -6,7 +6,7 @@
           <h1>{{ quiz.title }}</h1>
         </b-col>
       </b-row>
-      <b-row>
+      <b-row v-if="$store.getters.isAdmin">
```

Comment:
isAdmin if for the admin of the website. The quiz moderator is the person who creates the quiz. We may need to add a field `moderators: ['user1', 'user2']` in the quiz object and use that field instead of isAdmin. Look at the implementation of event admins in event video page